### ISSUE TITLE
Addon tab not shown in Storybook v7

### ISSUE THREAD

**Issue by vmilan** [pocka/storybook-addon-designs#171]
### Affected design types

- [X] figma
- [x] figspec
- [x] link
- [x] image
- [x] iframe

### Describe the bug

Hi, I'm testing Storybook v7 and the addon design tab is not shown.

Console log throws an error:

```
Uncaught TypeError: addons_default.register is not a function
    at register (index.js:9:12)
    at register.js:3:1
```

### How to reproduce the bug?

main.js:

```
  addons: [
    ...
    'storybook-addon-designs'
  ],
```

package.json:

```
    "storybook": "^7.0.0-alpha.34",
    "storybook-addon-designs": "^7.0.0-alpha.2",
    "@storybook/addon-actions": "^7.0.0-alpha.34",
    "@storybook/addon-essentials": "^7.0.0-alpha.34",
    "@storybook/addon-links": "^7.0.0-alpha.34",
    "@storybook/react": "^7.0.0-alpha.34",
    "@storybook/react-webpack5": "^7.0.0-alpha.34",
```

story:

```
import { withDesign } from 'storybook-addon-designs';

...

const options = {
  title: 'Component title',
  component: Component,
  argTypes: {
    color: {
      control: {
        type: 'select',
        options: ['opt1', 'opt2', ...]
      }
    }
  },
  decorators: [withDesign],
  parameters: {
    design: {
      type: 'figma',
      url: 'https://www.figma.com/file/.....'
    }
  }
};
export default options;
```

### Expected behaviour

To show the design tab as usual.

### Environment

- [X] Chrome / Chromium / Chromium based browsers
- [ ] Firefox
- [ ] macOS Safari
- [ ] iOS Safari
- [ ] Samsung Internet
- [ ] Other

### Affected versions

7.0.0-alpha.2

### Storybook versions

7.0.0-alpha.34"

**Comment by MichaelGregory** [comment]
Looks like the change has been made to support Storybook v7 in https://github.com/pocka/storybook-addon-designs/pull/160/. Just need to have a new version published.

**Comment by pocka** [comment]
Sorry I just forgot to create a new alpha release...

**Comment by pocka** [comment]
Fixed in [v7.0.0-alpha.3](https://github.com/pocka/storybook-addon-designs/releases/tag/v7.0.0-alpha.3). Please re-open if the problem still exists in the new version.

If you find another compatibility problem, please open a new issue.

**Comment by interphased** [comment]
I'm still having this issue in `Storybook 7.0.0-alpha.46` and `storybook-addon-designs 7.0.0-beta.0`.

```
index.js:10 Uncaught TypeError: addons_default.register is not a function
    at register (index.js:10:12)
    at register.js:3:1
```
